src/util/ellipse/calculateEllipseStatistics.test.js

```diff
@@ -1,18 +1,120 @@
 import calculateEllipseStatistics from './calculateEllipseStatistics.js';
 
 describe('calculateEllipseStatistics.js', () => {
-  // prettier-ignore
-  let fakePixels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
-                    0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
-  let fakeEllipseBounds = {
+  // Prettier-ignore
+  const fakePixels = [
```

Comment:
This is interesting. I wonder what tooling setup differences we have that cause this.
I use `lint-on-save`, and `// prettier-ignore` was preserving my formatting